components/Card.tsx

```diff
@@ -5,50 +5,23 @@ import { MemoryType } from '@/pages/memory-list'
 import CardPlayButton from './button/CardPlayButton'
```

Comment:
```suggestion
import CardPlayButton from './Button/CardPlayButton'
```
- 폴더명이 `button` 이네용!? 아마 컨벤션이 파스칼케이스라 대문자로 되어있을텐데 변경해주시면 감사합니다 👍
- 아마 전체적으로 저희 얘기했던걸로 변경될거라 나중에 한번에 하셔도 문제는 없을거 같습니다ㅎㅎ